tests/test_master.py

```diff
@@ -1,11 +1,17 @@
 from v6_carrier_py import master
 from unittest.mock import MagicMock
 import pytest
+import pandas as pd
 
 ID = 1
 COLUMN_NAMES = ['name1', 'name2']
```

Comment:
You could merge these with COLUMN1 and COLUMN2 I guess?

Reply:
Done